api/models/team.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const teamSchema = mongoose.Schema({
+	_id: mongoose.Schema.Types.ObjectId,
+	City: String,
+	Name: String
```

Comment:
Wondering if we can adopt camelCase naming (as that is generally the accepted convention in Javascript). I would expect a call to "/teams" to return something like:
{
    name: "Dolphins",
    city: "Miami",
    state: "FL"
}